### PR TITLE
Create ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61639](https://cumulocity.atlassian.net/browse/MTM-61639)
Original PR: [7748](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7748)

[MTM-61639]: https://cumulocity.atlassian.net/browse/MTM-61639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: load translations from dedicated app (#7748)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md]: The Claude AI suggested to use the following title:
```suggestion
title: Load translations from a dedicated application
```

File: content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61639
+version: 1021.17.0
+---
+load translations from dedicated app (#7748)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md]: The Claude AI suggested to use the following body:
```suggestion
To improve the maintainability and separation of concerns in the application, the translations are now loaded from a dedicated app. Previously, the translations were scattered across different parts of the codebase, making it difficult to manage and update them consistently. With this change, all translations are consolidated in a single location, simplifying the process of adding, updating, and maintaining translations. This change does not have any direct impact on end-users or the functionality of the application, but it contributes to a cleaner and more organized application infrastructure.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@reey Please review and approve

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61639
+version: 1021.17.0
+---
+To improve the maintainability and separation of concerns in the application, the translations are now loaded from a dedicated app. Previously, the translations were scattered across different parts of the codebase, making it difficult to manage and update them consistently. With this change, all translations are consolidated in a single location, simplifying the process of adding, updating, and maintaining translations. This change does not have any direct impact on end-users or the functionality of the application, but it contributes to a cleaner and more organized application infrastructure.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md]:
```suggestion
To improve the maintainability in the application, the translations are now loaded from a dedicated application. Previously, the translations were scattered across different parts of the codebase, making it difficult to manage and update them consistently. With this change, all translations are consolidated in a single location, simplifying the process of adding, updating, and maintaining translations. This change does not have any direct impact on end users or the functionality of the application, but it contributes to a cleaner and more organized application infrastructure.
```

File: content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Load translations from a dedicated application
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-17-0-load-translations-from-dedicated-app-7748.md]:
```suggestion
title: Translations loaded from a dedicated application
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**